src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -330,14 +330,14 @@
   {:render
    (fn [{:keys [props]}]
      (let [sep [:span {} " " (icons/font-icon {:style {:fontSize "50%"}} :angle-right) " "]
-           crumbs (:crumbs props)]
+           crumbs (filter some? (:crumbs props))]
```

Comment:
I need it because of the way I'm using it at the bottom of billing_management.cljs.  I conditionally add the second crumb, otherwise it's nil.